app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -1055,4 +1055,14 @@ private function decodeAmpersands(&$content)
             }
         }
     }
+
+    /**
+     * Calculate Read Percentage for each Email.
+     *
+     * @return int
+     */
+    public function getReadPercentage($includevariants = false)
+    {
+        return ($includevariants) ? round($this->getSentCount(true) > 0 ? ($this->getReadCount(true)/($this->getSentCount(true))*100) : 0, 2) : round($this->getSentCount() > 0 ? ($this->getReadCount()/($this->getSentCount())*100) : 0, 2);
```

Comment:
It's definitely an improvement, but could you expand the if statements to full
```
if (...) {
    ...
}
```
format so the readability would improve?

Another improvement could be if you'd pass the `$includevariants` variable to `$this->getSentCount($includevariants)` and `$this->getSentCount($includevariants)`. It should eliminate a half of the code.